docs/administration-guide/databases/bigquery.md

```diff
@@ -1,25 +1,111 @@
-
 ## Working with Google BigQuery in Metabase
 
-Starting in v0.15.0 Metabase provides a driver for connecting to BigQuery directly and executing queries against any datasets you have.  The below sections provide information on how to get connected and troubleshoot any issues that may come up.
+This page provides information on how to create and manage a connection to a Google [Bigquery](https://cloud.google.com/bigquery) dataset.
+
+> **Starting in v0.15.0**, Metabase provides a driver for connecting to BigQuery directly and executing queries against any datasets you have.  
+>**Starting in v0.36.0**, Metabase allows you to connect to your BigQuery dataset using a [Service Account](https://cloud.google.com/iam/docs/understanding-service-accounts).
```

Comment:
Because we version docs, we typically haven't included references to specific versions like this.

Reply:
The 0.15 notice was in the current docs, so I followed convention. But I agree: no need to disclose that info here.